includes.mk

```diff
@@ -0,0 +1,11 @@
+check-docker:
+	@if [ -z $$(which docker) ]; then \
+		echo "Missing \`docker\` client which is required for development"; \
+		exit 2; \
+	fi
+
+check-kubectl:
+	@if [ -z $$(which kubectl) ]; then \
+		echo "Missing \`kubectl\` client which is required for development"; \
```

Comment:
Agreed.  I think I'd like to revisit this after we have reasonable docs on hackin/contributing to v2.  I'm thinking the ideal message contains a link to the docs on setting up the development environment.